src/components/Common/ItemPicker.vue

```diff
@@ -0,0 +1,117 @@
+<template>
```

Comment:
Lets rename this component to ColumnPicker, similar as to how FieldPicker is named in Common in compose